Hw5_Hibernate/src/main/java/HomeWorks/HW5/hibernate/service/UserServiceImpl.java

```diff
@@ -0,0 +1,82 @@
+package HomeWorks.HW5.hibernate.service;
+
+import HomeWorks.HW5.hibernate.controller.dto.AddressDto;
+import HomeWorks.HW5.hibernate.controller.dto.UserDto;
+import HomeWorks.HW5.hibernate.dao.AddressRepository;
+import HomeWorks.HW5.hibernate.dao.UserDao;
+import HomeWorks.HW5.hibernate.dao.UserRepository;
+import HomeWorks.HW5.hibernate.dao.entities.AddressEntity;
+import HomeWorks.HW5.hibernate.dao.entities.UserEntity;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Objects;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+    private UserRepository userRepository;
+    private AddressRepository addressRepository;
+    private UserDao userDao;
+
+    @Autowired
+    public UserServiceImpl(UserRepository userRepository, UserDao userDao, AddressRepository addressRepository) {
+        this.userRepository = userRepository;
+        this.addressRepository = addressRepository;
+        this.userDao = userDao;
+    }
+
+    public UserDto createDto(UserEntity userEntity) {
```

Comment:
private method, if it is necessary only for class logic

Reply:
fixed